README.md

```diff
@@ -75,6 +75,10 @@ When VPC and VPC endpoint are created, please add following configuration to `AW
 
 [AWS documentation](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-api-endpointconfiguration.html) for endpoint configuration.
 
+### Uninstall the application with SAM CLI
+
+Run `sam delete` to delete the application. Note: S3 bucket and DynamoDB, created by the application, won't be deleted automatically and require to be removed manually.
```

Comment:
Let's move the `Note:` sentence to its own paragraph with bold/emphasis.

Reply:
ok, agree.